docs/devel/generating-clientset.md

```diff
@@ -48,7 +48,8 @@ the types (e.g., Pods) that you want to generate clients for with the
 namespace scoped (e.g., PersistentVolume), you need to append the
 `nonNamespaced=true` tag as well.
 
-- Running the client-gen tool: you need to use the command line argument
+- Running the client-gen tool: this is done [in a script](../../hack/update-codegen.sh).
```

Comment:
If you are generating client in the Kubernetes repository, you can add the group/version in (../../hack/update-codegen.sh) and run the script. Otherwise, you need to use the command line argument...